src/db/tables/subscribers.js

```diff
@@ -196,6 +196,34 @@ async function setOnerepProfileId (subscriber, onerepProfileId) {
     })
 }
 
+/**
+ * @param {number} onerepProfileId
+ * @param {number} oneRepScanId
+ */
+async function setOnerepScan (onerepProfileId, oneRepScanId) {
+  await knex('onerep_scans')
+    .insert({
+      onerep_profile_id: onerepProfileId,
+      onerep_scan_id: oneRepScanId,
+      created_at: knex.fn.now()
+    })
+}
```

Comment:
We're already using `Onerep`/`onerep_` in various places, I think it might be the winning convention, but I'll check. @Vinnl had an opinion on this IIRC

Reply:
Heh yeah, a soft opinion: I'd use a single capital letter for all external names, just for predictability in edge cases, e.g. `isRSSXML` vs `isRssXml`, `isJsonld` vs `isJsonLd` vs `isJSONLD`, or `getFxaProfile` vs `getFxAProfile` vs `getFXAProfile`. I won't die on that hill though :)